xds/src/test/java/io/grpc/xds/GrpcXdsClientImplDataTest.java

```diff
@@ -2456,6 +2465,61 @@ public void processCluster_parsesAudienceMetadata()
     assertThat(update.parsedMetadata()).isEqualTo(expectedParsedMetadata);
   }
 
+  @Test
+  public void processCluster_parsesAddressMetadata()
+      throws ResourceInvalidException, InvalidProtocolBufferException {
```

Comment:
Feel free in tests to just do `throws Exception`. The specific types don't generally matter much, because the test will fail anyway.